packages/select/src/Select.spec.tsx

```diff
@@ -113,6 +113,37 @@ describe('packages/select', () => {
     expect(getByTextFor(listbox, 'Explicit placeholder')).toBeVisible();
   });
 
+  test('when "usePortal" is not set, content is portaled to the end of the DOM', async () => {
```

Comment:
This is being tested by the popover component, and usePortal is just being passed through to it, right?

Reply:
Yeah but I thought it didn't hurt to add tests to make sure that the prop was properly passed from Select to ListMenu. Would you prefer that I remove these two tests?

Reply:
Yeah – if we were doing something different than just passing directly to portal, testing this would be fine, but this is one of the benefits of using our own components as building blocks – we can avoid testing the same implementation twice!

Reply:
Sure, removing now.